### PR TITLE
Feature/reporting log

### PR DESCRIPTION
**Pull request by Johanndutoit**
Adds a reporting table that can be used to list logs from the nodes. After updating this will send all steps up to the server that will allow teams to check back and see what the node is reporting as wrong.

### CODE REVIEW

**Comment by smn** [comment]
nice, let me know when this is ready for review.

**Comment by Johanndutoit** [comment]
@smn if your happy I'm good. The remaining .04% down is from a previous PR. Tests in this one would cover the entire endpoint.

**Comment by Johanndutoit** [comment]
@jonathanendersby @BarryBotha ^ thoughts ?

**Comment by smn** [comment]
@Johanndutoit where's the client side of this reporting living?

**Comment by smn** [comment]
@Johanndutoit happy for this to land ahead of #11.

How are these reports exposed in the UI? That'll be a separate PR?

**Comment by Johanndutoit** [comment]
@smn yes idea was a separate PR. Wanted to get the data working and storing as a start. And will need to land before #11 to ensure that the nodes are sending the data to a existing endpoint.

File: goddard/schema.coffee [hunk 1]
```diff
@@ -36,6 +36,15 @@ module.exports = exports = (app) ->
 	app.set('database', sequelize)
 
 	# set the models
+	Models.reports = sequelize.define('reports', {
+
+		nodeId: { type: Sequelize.INTEGER, field: 'nodeId' }
+		status: { type: Sequelize.STRING(255), field: 'status' }
+		message: { type: Sequelize.STRING(255), field: 'message' }
```
Comment (smn) [inline goddard/schema.coffee:51]: timestamps please @Johanndutoit !

**Comment by smn** [comment]
Sorry, just spotted the lack of timestamps in the schema. Please add before we land, otherwise these reports are useless.

File: goddard/schema.coffee (hunk 1 above)
Comment (Johanndutoit) [inline goddard/schema.coffee:51]: part of the ORM. Will be _createdAt_ and _updatedAt_. But will add the _timestamp_ from the file too so you have it all.

Comment (Johanndutoit) [inline goddard/schema.coffee:51]: Actually the timestamps from the file would be useless as if it fails you won't see that timestamp again so createdAt and updatedAt alone are better.

Comment (smn) [inline goddard/schema.coffee:51]: cool

**Comment by smn** [comment]
:+1:

**Comment by smn** [comment]
There was a merge conflict on the schema, I resolved it but @Johanndutoit I'd appreciate it if you could review.

**Comment by Johanndutoit** [comment]
@smn sure give me a sec

**Comment by smn** [comment]
Nevermind, the schema's correct. I had to chose both the schema for the tokens & the reports.

**Pull request merged**